### ISSUE TITLE
[iOS][macOS] `TabView.ContainerChanging` called more than necessary

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#4949]
## Current behavior

While on other platforms the TabViewListView ContainerChanging event fires only once per item, on iOS it is more.

## Expected behavior

The event should be triggered only once per item.

## How to reproduce it (as minimally and precisely as possible)

Uncomment `TabViewTests.VerifyItemsAreCreatedOnlyOnce` and run the runtime test on iOS.

## Environment

Nuget Package:

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [x] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [x] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by MartinZikmund** [comment]
No longer relevant, works correctly for Skia renderer targets
